src/createSchemaForm.tsx

```diff
@@ -462,9 +460,7 @@ export function createTsForm<
         <ActualFormComponent {...formProps} onSubmit={submitFn}>
           {renderBefore && renderBefore({ submit: submitFn })}
           {CustomChildrenComponent ? (
-            <CustomChildrenComponent
-              renderedFields={renderedFields}
-            ></CustomChildrenComponent>
+            <CustomChildrenComponent {...renderedFields} />
```

Comment:
If we want to pass more props to this component later does it seem ok for them to be mixed in with the fields? Only concern I see is name collision, probably not a big worry though

Reply:
Yeah that's a good point but I can't think of anything else we'd want to pass here at the moment (not saying there isn't anything though) @scamden

if we end up needing it we can change it back it 2.0 😄

I really like having the API as simple as possible though, and I think not having to unpack is a little more dev friendly

Reply:
Agreed :)